src/js/api.js

```diff
@@ -0,0 +1,22 @@
+//  marvel api
+const url = 'https://gateway.marvel.com/v1/public/characters?ts=1&apikey=1b4878691fe6328b46c8b8eb4bde2f9f&hash=93ebee6185894b7ebc29d9ed94f988a9&limit=100&offset=70';
+
+const characterUrl = 'https://gateway.marvel.com/v1/public/characters';
+
+// get the data
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Please don't disable `eslintrc`, kindly try and solve the error manually. 
-  I am requesting this because `eslintrc. js` is a configuration file for a tool named ESLINT. ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs

Reply:
Done!